app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaScreen.kt

```diff
@@ -0,0 +1,207 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material3.Divider
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.rememberPagerState
+import com.wire.android.R
+import com.wire.android.model.Clickable
+import com.wire.android.model.SnackBarMessage
+import com.wire.android.model.UserAvatarData
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.progress.WireCircularProgressIndicator
+import com.wire.android.ui.common.snackbar.SwipeDismissSnackbarHost
+import com.wire.android.ui.common.topappbar.WireCenterAlignedTopAppBar
+import com.wire.android.ui.common.topappbar.search.SearchBarState
+import com.wire.android.ui.common.topappbar.search.SearchTopBar
+import com.wire.android.ui.common.topappbar.search.rememberSearchbarState
+import com.wire.android.ui.home.conversationslist.common.ConversationList
+import com.wire.android.ui.home.conversationslist.model.ConversationFolder
+import com.wire.android.ui.home.newconversation.common.SendContentButton
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.util.extension.getActivity
+import kotlinx.collections.immutable.persistentMapOf
+import kotlinx.coroutines.flow.SharedFlow
+
+@Composable
+fun ImportMediaScreen(importMediaViewModel: ImportMediaViewModel = hiltViewModel()) {
+    val context = LocalContext.current
+    val searchBarState = rememberSearchbarState()
+    val snackBarHostState: SnackbarHostState = remember { SnackbarHostState() }
```

Comment:
We do not use it here in this function, so I think it is better to extract this into ImportMediaContent instead